maskit/optimizers.py

```diff
@@ -1,5 +1,165 @@
+from typing import Optional, Tuple
 from pennylane import GradientDescentOptimizer, AdamOptimizer
+from pennylane._grad import grad as get_gradient
 from enum import Enum
+from pennylane.numpy import ndarray
+import scipy.optimize as sciopt
+
+
+class L_BFGS_B:
+    """
+    The L-BFGS-B optimiser provides a wrapper for the implementation provided
+    in scipy. Please see the appropriate documentation for further details.
```

Comment:
This should link to the docs (if you use intersphinx).
```suggestion
    in scipy. Please see the :py:func:`~sciopt.fmin_l_bfgs_b` documentation for further details.
```